docs/install.rst

```diff
@@ -434,9 +434,14 @@ With these installed, the documentation can be built with the
 
     python setup.py build_sphinx
 
-This can be done **without** building Sherpa (either an installation
-or development version), since Mock objects are used to represent
-compiled and optional components.
+Note that this uses the installed version of sherpa, so if you want to make
```

Comment:
I think this section needs a minor rework because

a) I can't seem to use `build_sphinx` any more
b) we know `build_sphinx` has been deprecated so may as well remove it
c) at the start of the `building the documentation` section we currently say

```
Building the documentation requires the Sherpa source code and several
additional packages:
```

and we need to update it to mention being able to build Sherpa - so smoething like

```
Building the documentation requires being able to build Sherpa
(:ref:`build-from-source`) and several additional packages:
```

[I haven't run this through sphinx so I'm not sure it will look]

Reply:
I note that this is also addressed in #1608 although that version still talks about mock objects, so there will be a small clash once one of these is merged.